src/features/import/store.ts

```diff
@@ -1,16 +1,55 @@
-import { Column, ImportedFile } from './utils/types';
+import {
+  Column,
+  IMPORT_ERROR,
+  ImportedFile,
+  PersonImport,
+  PersonImportSummary,
+} from './utils/types';
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
 export interface ImportStoreSlice {
+  importResult: PersonImport;
+  importErrors: IMPORT_ERROR[];
+  preflightSummary: PersonImportSummary;
   pendingFile: ImportedFile;
 }
 
+const emptySummary: PersonImportSummary = {
+  addedToOrg: {
+    byOrg: {},
+    total: 0,
+  },
+  created: {
+    total: 0,
+  },
+  tagged: {
+    byTag: {},
+    total: 0,
+  },
+  updated: {
+    byField: {},
+    total: 0,
+  },
+};
+
 const initialState: ImportStoreSlice = {
+  importErrors: [],
+  importResult: {
+    accepted: '',
+    completed: null,
+    report: {
+      person: {
+        summary: emptySummary,
+      },
+    },
+    status: null,
+  },
   pendingFile: {
     selectedSheetIndex: 0,
     sheets: [],
     title: '',
   },
+  preflightSummary: emptySummary,
```

Comment:
I feel like this should also be `null` until a summary has been generated (post preflight).

Reply:
good point also!